src/layoutcomponent/Navbar.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Logo } from "./Logo";
+import { NavbarSearch } from "./NavbarSearch";
+import { HamburgerSearch } from "./HamburgerSearch";
+import "../styles/Navbar.css";
+import { UserProfileInNavbar } from "./UserProfileInNavbar";
+import { HamburgerNavbar } from "./HamburgerNavbar";
+import { useState } from "react";
+
+const Navbar = () => {
+  const [showHamburgerMenu, setShowHamburgerMenu] = useState(false);
+  const displayHamburger = () => {
+    setShowHamburgerMenu((prev) => true);
+  };
+  return (
+    <>
+      <HamburgerNavbar
+        showHamburgerMenu={showHamburgerMenu}
+        setShowHamburgerMenu={setShowHamburgerMenu}
+      ></HamburgerNavbar>
+      <nav class="apnatheka_navbar">
```

Comment:
change class to className in this file as well

Reply:
sure